Src/WitsmlExplorer.Frontend/hooks/useExport.ts

```diff
@@ -0,0 +1,41 @@
+export interface ExportOptions {
+  outputMimeType: string;
+  fileExtension: string;
+  separator: string;
+  newLineCharacter: string;
+  omitSpecialCharactersFromFilename?: boolean;
+  appendTimeStamp?: boolean;
+}
+interface ExportReturn {
```

Comment:
anyone knows a good name for this interface?

Reply:
What about ExportDataForDownload?

Reply:
What about ExportObject ?

Reply:
sounds good

Reply:
renamed in latest commit